UI/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do we need that?

Reply:
No, it was generated adding Cypress schematic to Angular.

Reply:
👍